vimeo-networking/src/main/java/com/vimeo/networking/model/VimeoAccount.java

```diff
@@ -39,81 +40,156 @@
 public class VimeoAccount implements Serializable {
 
     private static final long serialVersionUID = -8341071767843490585L;
-    //    private static final String TOKEN_TYPE_BEARER = "bearer";
 
+    @Nullable
     @GsonAdapterKey("access_token")
-    public String accessToken;
+    public String mAccessToken;
```

Comment:
Yes, it was recognized when stag was released and is listed in the future enhancements section of its readme. Also, it is ticketed for us to take on here: https://vimean.atlassian.net/browse/VA-1926